sw/vendor/riscv_compliance/riscv-target/opentitan/device/rv32imc/handler.S

```diff
@@ -0,0 +1,38 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+.section .text.trap;
+.align  4;
+
+_trap_start:
+    j _trap_exception
+    j _int_exc
+    j _int_exc
+    j _int_exc
+    j _int_exc
+    j _int_exc
+    j _int_exc
+    j _int_exc
+    j _int_exc
+    j _int_exc
+    j _int_exc
+    j _int_exc
+    j _int_exc
```

Comment:
Can you add a comment explaining why you've filled these 12 words with jumps? I can imagine why, but 12 seems awfully specific.

Reply:
:) 
let me dig a bit as to what this handler is doing. I copied it from ibex, which may have been copied from somewhere else

Reply:
i am baffled by this... do you have any explanation? The best I have is that this is creating some kind of alignment?
Because if we jump to `_trap_exception`, won't we end up in something that looks like _int_exc anyways?

I also don't understand why there's a separate `1:` label. Anyhow, I took a bunch of stuff out and I'm curious how it will run now.